src/main/java/com/lohika/morning/ecs/domain/attendee/AttendeeRepositoryImpl.java

```diff
@@ -0,0 +1,29 @@
+package com.lohika.morning.ecs.domain.attendee;
+
+
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+
+@Repository
+public class AttendeeRepositoryImpl implements AttendeeRepositoryCustom {
+  @PersistenceContext
+  private EntityManager entityManager;
+
+  public List<Attendee> filterBy(List<String> strings) {
```

Comment:
Isn't it possible just to declare a method in AttendeeRepository like this:
`findByfirstNameLikeOrSecondNameLikeOrEmailLike(firstName, lastName, email)`?